flowzone.yml

```diff
@@ -67,6 +67,7 @@
       environment: ${{ inputs.balena_environment }}
       fleet: ${{ matrix.slug }}
       source: ${{ inputs.working_directory }}
+      note: ${{ steps.release_notes.outputs.note }}
```

Comment:
the only side effect I found is the addition of extra quotes around the note:

<img width="457" alt="image" src="https://github.com/product-os/flowzone/assets/2033996/e4744a03-be89-40e1-83df-30e003930ae3">

Reply:
is that coming from the DtB action or Flowzone? We should identify the fix as it's pretty ugly IMO

Reply:
https://github.com/balena-io/deploy-to-balena-action/pull/341